libs/nest/swagger/README.md

```diff
@@ -0,0 +1,7 @@
+# Nest Swagger
```

Comment:
Maybe add a bit of a description in the `README` what this is and does, and how it should be used with few code samples.